src/pybind/mgr/cephadm/module.py

```diff
@@ -2241,6 +2265,7 @@ def _generate_prometheus_config(self):
 """.format(
     mgr_scrape_list=str(mgr_scrape_list),
     node_configs=str(node_configs),
+    alertmgr_configs=str(alertmgr_configs)
```

Comment:
`alertmgr_configs` is an empty string at this point

Reply:
yep, it’s a typo, I’ll fix it